tests/runtime/probe

```diff
@@ -256,6 +256,11 @@ PROG interval:s:1 { exit(); } END { printf("end"); }
 EXPECT end
 TIMEOUT 2
 
+NAME BEGIN,END
+PROG BEGIN,END { printf("Hello\n"); exit(); }
```

Comment:
```suggestion
PROG BEGIN,END { print(probe); exit(); }
```

maybe better? That way youre sure you didnt trigger BEGIN twice or swapped ordering

Reply:
ah good point :)